### PR TITLE
add configurable throughput + client params to http logs

### PR DESCRIPTION
**Pull request by OVI3D0**
### Description
cherry-pick changes for http-logs onto branch 7

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: http_logs/test_procedures/default.json [hunk 1]
```diff
@@ -59,76 +59,50 @@
...
-          {%-if search_clients is defined and search_clients %}
-          ,"clients": {{ search_clients | tojson}}
-          {%- endif %}
+          "iterations": 100,
+          "target-throughput": {{ status_400s_in_range_target_throughput or target_throughput | default(50) | tojson }},
+          "clients": {{ status_400s_in_range_search_clients or search_clients | default(1) }}
         },
         {
           "operation": "hourly_agg",
+          "warmup-iterations": 50,
```
Comment (IanHoang) [inline http_logs/test_procedures/default.json:99]: I think this was changed from `100` to `50` in the process of this cherry-pick ([Reference](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/7/http_logs/test_procedures/default.json#L132)). Is there a reason why it was changed?

Comment (OVI3D0) [reply http_logs/test_procedures/default.json:99]: No, may have been a bug in my script or the cherry-pick. I can put out a quick fix!

File: http_logs/test_procedures/default.json [hunk 2]
```diff
@@ -59,87 +59,54 @@
...
+          "iterations": 100,
+          "target-throughput": {{ hourly_agg_target_throughput or target_throughput | default(0.2) | tojson }},
+          "clients": {{ hourly_agg_search_clients or search_clients | default(1) }}
+        },
+        {
+          "operation": "scroll",
+          "warmup-iterations": 100,
+          "iterations": 100,
+          "target-throughput": {{ scroll_target_throughput or target_throughput | default(0.2) | tojson }},
+          "clients": {{ scroll_search_clients or search_clients | default(1) }}
```
Comment (IanHoang) [inline http_logs/test_procedures/default.json:109]: We should remove this and preserve lines 112 -117

**Review by IanHoang — APPROVED** [review]
Looks good to me 👍

**Pull request merged**